src/components/tabs/tabs.md.scss

```diff
@@ -54,14 +54,14 @@ $tabs-md-tab-text-transition:                 transform .3s ease-in-out !default
 /// @prop - Transform for the active tab button icon when the layout is icon-top, icon-only, or title-only
 $tabs-md-tab-icon-transform-active:           translate3d(0, -2px, 0) !default;
 
-/// @prop - Transform for the active tab button icon when the layout is icon-right
-$tabs-md-tab-icon-right-transform-active:     translate3d(2px, 0, 0) !default;
+/// @prop - Transform for the active tab button icon when the layout is icon-end
+$tabs-md-tab-icon-end-transform-active:     translate3d(2px, 0, 0) !default;
 
 /// @prop - Transform for the active tab button icon when the layout is icon-bottom
 $tabs-md-tab-icon-bottom-transform-active:    translate3d(0, 2px, 0) !default;
 
-/// @prop - Transform for the active tab button icon when the layout is icon-left
-$tabs-md-tab-icon-left-transform-active:      translate3d(-2px, 0, 0) !default;
+/// @prop - Transform for the active tab button icon when the layout is icon-start
+$tabs-md-tab-icon-start-transform-active:      translate3d(-2px, 0, 0) !default;
```

Comment:
same here! it is not aligned hahha sorry :D

and I think this is breaking change too, we want to avoid them:
what if we do:
```
// deprecated
$alert-left-label:            7px !default;
/// @prop - 
$alert-start-label:          $alert-left-label !default;

alert-label {
  left: $alert-start-label;
}
```